test/test_imu/getExpectedResults.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
```

Comment:
What does this file do?

Reply:
In my TestIMU, I give the filter some inputs and I want to validate the outputs. I am not going to sit here and do hand calcs for an hour to get the wrong answer so it just uses a python library that has implemented the filter to find the expected results to validate my implementation was correct.